sql/sql_show.cc

```diff
@@ -3778,8 +3778,11 @@ class Add_status : public Do_THD_Impl {
 void calc_sum_of_all_status(System_status_var *to) {
   DBUG_TRACE;
   mysql_mutex_assert_owner(&LOCK_status);
-  /* Get global values as base. */
+
+  /* Get global values as base but net_buffer_length. */
   *to = global_status_var;
+  to->net_buffer_length = 0;
+
```

Comment:
Is it needed? It looks like we do not modify global_status_var.net_buffer_length value anywhere. It should always be zero.

If it is needed, we should do the same in PFS_connection_iterator::visit_global() after capturing global_status_var.net_buffer_lenght. 

How to test this path? It is not the path we go when we execute 'show global status like 'net_buffer_length';' 
mysqladmin?

This change also causes change of the client-server protocol (additional field to be sent). Is it backward compatible?

Reply:
I guess it is not necessary, I will remove it. I will add a couple of tasks to test this path and check the backward compatibility

Reply:
@kamil-holubicki, I do not know if it is possible to test this path. The reason is that `calc_sum_of_all_status` is just used in <https://github.com/ldonoso/percona-server/blob/7b919163899d5aa2ebf1b7703d3393e56899a293/sql/sql_parse.cc#L2126-L2140>

Please, note that just the status variables `long_query_count` and `opened_tables` are used.

Reply:
@kamil-holubicki, It'd say it is backward compatible. The status vars are [modelled as a table](https://github.com/ldonoso/percona-server/blob/7b919163899d5aa2ebf1b7703d3393e56899a293/storage/perfschema/table_global_status.h#L84).

From the point of view of the client/server protocol, returning a new row in a table
should not be a breaking change.